Wrappers/d3d8/IDirect3DDevice8.cpp

```diff
@@ -1334,6 +1334,12 @@ HRESULT m_IDirect3DDevice8::DrawScaledSurface()
 		ProxyInterface->SetRenderTarget(pBackBuffer, pStencilBuffer);
 	}
 
+	// Draw Overlays
+	OverlayRef.DrawOverlays(ProxyInterface, Desc.Width * ScaleFactor, Desc.Height * ScaleFactor);
```

Comment:
This concerns me.  If we are sending `DrawOverlays` in a different resolution from the last time we called it `Desc.Width` vs `Desc.Width * ScaleFactor` that causes it to reinitialize the overlay.  Since we are doing this every frame, we are causing it to reinitialize the overlay every frame.  I don't think that is a good idea.

Reply:
Code was removed in the last commit!